assets/src/edit-story/components/library/panes/media/local/mediaPane.js

```diff
@@ -17,7 +17,7 @@
 /**
  * External dependencies
  */
-import { createRef, useCallback } from 'react';
+import { createRef, useCallback, useMemo } from 'react';
```

Comment:
We may remove `useMemo` if not being used.

Reply:
Yes, we're using the `useMemo` in line no 71.